src/engraving/rw/write/writer.h

```diff
@@ -27,8 +27,8 @@
 #include "writecontext.h"
 #include "../compat/writescorehook.h"
 
-namespace mu::engraving::rw400 {
-class Write400 : public rw::IWriter
+namespace mu::engraving::write {
+class Writer : public rw::IWriter
```

Comment:
No, I don't think so, because:
* The `rw` namespace contains the public interfaces to be used. And in subfolders there are internal implementations that do not need to be used outside (they are still used now, but this is just not a refinement and will be fixed in the future). The `IWrite` interface is intended for external use (relative to the read/wite subsystem).
* Now it is done uniformly with reading, the difference is only in the number of implementations of reading and writing.